plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/Actor.java

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.json;
+
+import java.util.Map;
+
+/**
+ * Represents Actor field of {@link Event}.
+ *
+ * @author Mykola Morhun
+ */
+public class Actor {
+    private String ID;
+    Map<String,String> Attributes;
```

Comment:
It is not common for java to use names of var or field with leading upper-case symbol